libraries/raw-rs/src/postprocessing/convert_to_rgb.rs

```diff
@@ -0,0 +1,36 @@
+use crate::Image;
+
+pub fn convert_to_rgb(mut image: Image<u16>) -> Image<u16> {
+	let Some(rgb_to_camera) = image.rgb_to_camera else { return image };
```

Comment:
It is incorrect. I had initially not used an Result when I added my first step and kept it that way in the subsequent steps. I will have to change it in the future. Although I will have to think how this will translate to the Node Graph. As far as I know the Node Graph doesn't automatically unwrap Option and Result.

Reply:
We can worry about Graphite integration details when implementing the nodes. Let's just focus on the API design here for the library itself.

I should also point out that the same question applies to these functions, where I also changed the `if let` into a `let else` to early return the original value, but it's worth exploring whether `None` or `Err` is a more appropriate return value in those cases.

- `pub fn gamma_correction()`
- `pub fn calculate_conversion_matrices()`
- `pub fn scale_colors()`

Reply:
Actually let's save this for a followup PR. I'll merge this one now.